idunn/blocks/wikipedia.py

```diff
@@ -100,6 +101,50 @@ def get_title_in_language(self, title, source_lang, dest_lang):
 
         return None
 
+class Wikidata:
+    _wiki_es = None
+    _es_lang = None
+
+    @classmethod
+    def get_wiki_index(cls, lang):
+        if cls._es_lang is None:
+            from app import settings
+            cls._es_lang = settings['ES_WIKI_LANG'].split(',')
+        if lang in cls._es_lang:
+            return "wikidata_{}".format(lang)
+        return None
+
+    @classmethod
+    def get_wiki_es(cls):
+        if cls._wiki_es is None:
+            from app import settings
+            cls._wiki_es = Elasticsearch(settings['WIKI_ES'])
```

Comment:
I don't think WIKI_ES should be a mandatory peace of our architecture.

It's quite specific to qwant internal architecture and it would be sad if it made it impossible to use idunn outside qwant

Reply:
indeed,
done